app/components/UI/Ramp/buy/hooks/useCryptoCurrencies.ts

```diff
@@ -0,0 +1,76 @@
+import { useRampSDK } from '../../common/sdk';
+import useSDKMethod from '../../common/hooks/useSDKMethod';
+import { useEffect, useMemo } from 'react';
+import { NATIVE_ADDRESS } from '../../../../../constants/on-ramp';
```

Comment:
nit: sort imports placing relative imports last
```suggestion
import { useEffect, useMemo } from 'react';
import useSDKMethod from '../../common/hooks/useSDKMethod';
import { useRampSDK } from '../../common/sdk';
import { NATIVE_ADDRESS } from '../../../../../constants/on-ramp';```